src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/release-please-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "release-type": "python",
+  "changelog-type": "github",
```

Comment:
I have waffled back and forth a few times, but this should be `default` - I'll be updating this value on other repos shortly.

Reply:
@nick-marnik sorry which key should be assigned with default value?

Reply:
bump